docs/core/install.md

```diff
@@ -51,12 +57,14 @@ to install other packages you can do so by:
 pip install fairchem-{package-to-install}
 ```
 
-## Dev install
+## Development install
 
-If you plan to make contributions you will need to clone (for windows user please see next section) the repo and install
+If you plan to make contributions you will need to fork and clone (for windows user please see next section) the repo, set up the environment, and install from source.
```

Comment:
"and install `fairchem-core` from source in editable mode with dev dependencies."